botocore/loaders.py

```diff
@@ -153,7 +154,10 @@ def load_file(self, file_path):
         full_path = file_path + '.json'
         if not os.path.isfile(full_path):
             return
-        with open(full_path) as fp:
+
+        # By default the file will be opened with locale encoding on Python 3.
+        # We specify "utf8" here to ensure the correct behavior.
+        with io.open(full_path, encoding='utf8') as fp:
```

Comment:
Usually in the code we use ``utf-8`` instead of ``utf8``. It would be nice to stay consistent. I do not think there is any drawbacks in using ``utf8`` over ``utf-8`` though.